VideoWeb/VideoWeb/ClientApp/src/app/services/conference/local-storage.service.ts

```diff
@@ -8,7 +8,7 @@ export class LocalStorageService {
     private loggerPrefix = '[LocalStorageService] -';
     constructor(private logger: LoggerService) {}
 
-    save<T extends object>(key: string, value: T, overwrite: boolean = true): boolean {
+    save(key: string, value: string | object, overwrite: boolean = true): boolean {
```

Comment:
Just trying to understand the logic here; is this change because you want to store a string a value? But because string is a primitive type in js, its not a type of obect?